pallets/pool-fees/src/lib.rs

```diff
@@ -772,6 +789,13 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		// Returns all fees of a pool divided by the buckets
+		pub fn get_pool_fees(pool_id: T::PoolId) -> Vec<(PoolFeeBucket, Vec<PoolFeeOf<T>>)> {
+			PoolFeeBucket::iter()
+				.map(|bucket| (bucket, ActiveFees::<T>::get(pool_id, bucket).into_inner()))
```

Comment:
Should it be divided or the divided result is already in `ActiveFees`?

Reply:
We want to divide the fees by the bucket, see [slack conversation](https://kflabs.slack.com/archives/C06F73K3XFD/p1707407253603519?thread_ts=1707130169.808409&cid=C06F73K3XFD).

Reply:
Thanks!